analyzers/its/sources/Net5/Net5/S4433.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.DirectoryServices;
+
+namespace Net5
+{
+    public class S4433
```

Comment:
This one doesn't have json (yet)

Reply:
It was in JSON. May be you've wanted to add something else?